### ISSUE TITLE
pluto.tv: breakage

### ISSUE THREAD

**Issue by Chasapple4** [uBlockOrigin/uAssets#25709]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I am not using uBO Lite along with other content blockers.
- [X] I have verified that the issue still exists after using the [`Optimal` or `Complete` filtering mode](https://user-images.githubusercontent.com/585534/195342612-85d109d9-9006-4eb5-95a5-fec8a4f233ea.png).
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBOL Lite and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://pluto.tv/us/live-tv/61b3b8c645b45d0007a9bb8d`
Or any channel on the live tv

### Category

breakage

### Description

Site fails to start a stream, only complete breaks playback

### Other extensions used

nothing else is an ad blocker or tracker blocker

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

```yaml
version: uBOL 2024.10.14.189
mode: complete / complete
rulesets:
 default
 isr-0
```

**Comment by stephenhawk8054** [comment]
What do you mean by "breaks playback"? Can you screenshot and describe in more details?

**Comment by Chasapple4** [comment]
There is nothing to screenshot as playback fails can't play anything on the Live TV section

**Comment by stephenhawk8054** [comment]
Then I can't reproduce the issue. Video plays fine for me
![image](https://github.com/user-attachments/assets/eccdd1af-d8e3-41c6-903c-ce8fde7c13d9)

**Comment by Chasapple4** [comment]
I can reproficed it 100% in both Lite and non Lite UBo

**Comment by Chasapple4** [comment]
I have reproduced it in Edge and VIvaldi

**Comment by stephenhawk8054** [comment]
Since I can't reproduce the issue, I can't investigate more.

**Comment by Chasapple4** [comment]
Even tho I gave in the initial report that it is broken?

**Comment by stephenhawk8054** [comment]
I mean if I can't reproduce the issue, how can I proceed to investigate on my side?

**Comment by Chasapple4** [comment]
Did you set Lite to Complete like I mentioned in the initial post?

**Comment by Chasapple4** [comment]
I have never seen it work, was something changed and not mention in this ticket?

**Comment by stephenhawk8054** [comment]
> Did you set Lite to Complete like I mentioned in the initial post?

Yes
![image](https://github.com/user-attachments/assets/462121ef-7cce-4c90-a783-8a09e20de261)

![image](https://github.com/user-attachments/assets/15eed0a3-e2dc-42ba-8513-0f2bc898d5e5)

> I have never seen it work, was something changed and not mention in this ticket?

Nothing has changed. I use the same version as yours. I originally could not reproduce your issue when you first reported few days ago, hence the asking for screenshots.

**Comment by Chasapple4** [comment]
I have it and seeing the thing breaking playback right now, as I had it on one channel and a few minutes later it was switch to another one (this is in private window), and the console (dev tools) is filling up with the content blocker blocking the video content (this is one of the sites I think using the ad server to also stream the main content)

**Comment by stephenhawk8054** [comment]
As I said

> Since I can't reproduce the issue, I can't investigate more.

**Comment by Chasapple4** [comment]
Are you using a beta version of Chrome?

**Comment by stephenhawk8054** [comment]
No, stable version, chromium `130.0.6723.59`

**Comment by Chasapple4** [comment]
Never seen the beak on any Chrome GUI set up

**Comment by Chasapple4** [comment]
Have you tried any other channel but the default one? (seems that one is streamed a bit different)

**Comment by Chasapple4** [comment]
<img width="1299" alt="image" src="https://github.com/user-attachments/assets/b92670cc-3793-40db-9098-c8f2b416651e">

**Comment by stephenhawk8054** [comment]
You can go to Network tab to see which connections are blocked by which connections

**Comment by Chasapple4** [comment]
100% something has been changed since I first reported the issue and it is not mentioned in this bug report

Pluto.tv does not use recently standards for streaming content that using an old m3u style playlist and the ads are injected into the same server from what I have seen, I had to disable UBo on the site as playback fails and has for a few years.  
![Screenshot 2024-10-19 215325
[pluto.tv-1729396397688.log](https://github.com/user-attachments/files/17448671/pluto.tv-1729396397688.log)
](https://github.com/user-attachments/assets/9a9fa7f4-c8d6-4a08-949b-3080b807ee57)

Also just reproduced it again while typing this

**Comment by Chasapple4** [comment]
Did the .log file not go through?

**Comment by stephenhawk8054** [comment]
uBOL is a static extension. There are no external connections to update the filter lists like uBO. Everything in a version is packaged the same for everyone until next version.

**Comment by stephenhawk8054** [comment]
> I had to disable UBo on the site as playback fails and has for a few years.

So, this could be related to ads not able to play. Do you have any browsers that still have uBO (not uBOL)?

**Comment by Chasapple4** [comment]
Yes, Vivaldi I have both extensions installed, UBo I have had it off on pluto.tv for years as it breaks playback, Lite is doing the same thing tho seems to be delayed.
Did my logs not get uploaded? (log from the console)

**Comment by Chasapple4** [comment]
SIDE NOTE
UBo blocks a zero day exploit in Chromium that has been there since version 1 and has been public info since 2009

**Comment by stephenhawk8054** [comment]
I don't see any logs.

> UBo I have had it off on pluto.tv for years as it breaks playback

Can you create a new browser profile, only install uBO with default settings there and confirm the issue is there first? Then we'll test on that profile.

Also turn off browser's built-in blocker there too.

**Comment by mapx-** [comment]
@Chasapple4 

- disable vivaldi blocker, disable edge Tracking prevention 
- if you have some other blocking software in your system just be sure there is no web shields enabled in such software
-

**Comment by Chasapple4** [comment]
it is 100% not complete

**Comment by Chasapple4** [comment]
I have reproducing it for years

**Comment by stephenhawk8054** [comment]
Please do this

> Can you create a new browser profile, only install uBO with default settings there and confirm the issue is there first? Then we'll test on that profile.
> 
> Also turn off browser's built-in blocker there too.

**Comment by Chasapple4** [comment]
@stephenhawk8054  I did that years ago and I have to exclude the site in both UBo and Lite otherwise it fails to play or breaks after a few minutes

**Comment by stephenhawk8054** [comment]
**Then we'll test on that profile.**

**Comment by stephenhawk8054** [comment]
If you cannot follow what we said, we can't help more, sorry. Good luck.

**Comment by Chasapple4** [comment]
This is 100% reproducible as I stated and has been for years

**Comment by Chasapple4** [comment]
Reproduced it on 2 different machines in 3 different browsers

**Comment by Chasapple4** [comment]
Reopen it if you refuse to investigate it, as I have reproduced it for years as I have said and even sent screenshots and logs

**Comment by Chasapple4** [comment]
https://github.com/uBlockOrigin/uAssets/issues/25709#issuecomment-2424541928 this is a screenshot that was asked for and yes Edge sat like that for about 8 hours while also stuck at 43% CPU usage for that tab while it was stuck due to Ublock origin lite blocking the main content

The trackers are not part of the video player like the ads are

**Comment by Chasapple4** [comment]
@MasterKia I have reproduced this for years, and tried it on a browser I rarely use and tried it and the site in there for the first time, (it works fine in Edge with the tracking protection as that does not touch ads), tho the logs and screenshot I provided have been ignored.

A private windows is pretty much a new profile as after that window is closed any session related content and anything done in the private window is gone.

Again I have reproduced it in 3 different browsers on 2 different machines

**Comment by preservehealth** [comment]
@liamengland1 got some time to take a look?

**Comment by Chasapple4** [comment]
Why are you ignoring the logs and screenshot I sent?    It is 100% reproducible and has been for years with both Lite and non Lite version of UBo.
As I said I don't use one if the browsers for the site so a new profile would do nothing. 

Who is making false spam reports?
